docs/quickstart.rst

```diff
@@ -400,6 +399,11 @@ For automated backups the passphrase can be specified using the
     A backup inside of the backup that is encrypted with that key/passphrase
     won't help you with that, of course.
 
+    In case you lose your repository, but have an older copy of it to restore
+    from, don't use that later for creating new backups – you would run into
+    security issues (reuse of nonce counter values). It is better to initialize
+    a new Borg repository.
```

Comment:
Guess this is only true if you also lost the clientside counter value at the same time.

Reply:
Added "and security info" to "lose your repository".